lib/GenericsAPI/Utils/MatrixUtil.py

```diff
@@ -19,7 +19,9 @@ def log(message, prefix_newline=False):
     time_str = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(time.time()))
     print(('\n' if prefix_newline else '') + time_str + ': ' + message)
 
+
 MATRIX_TYPE = ['ExpressionMatrix', 'FitnessMatrix', 'DifferentialExpressionMatrix']
+TYPE_ATTRIBUTES = {'description', 'scale', 'row_normalization', 'col_normalization'}
```

Comment:
should it also include 'col/row_attributemapping_ref' and 'diff_expr_matrix_ref'?

Matrix downloader puts those refs in metadata sheet.